### PR TITLE
.error notices in network/themes

### PR DESCRIPTION
**Pull request by joedolson**
Add `wp_admin_notice()` in `wp-admin/network`

Trac ticket: https://core.trac.wordpress.org/ticket/57791

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Review by costdev — APPROVED** [review]
All good to go with this one! :+1:

File: src/wp-admin/network/themes.php [hunk 1]
```diff
@@ -135,11 +135,25 @@
 				<div class="wrap">
 				<?php if ( 1 === $themes_to_delete ) : ?>
 					<h1><?php _e( 'Delete Theme' ); ?></h1>
-					<div class="error"><p><strong><?php _e( 'Caution:' ); ?></strong> <?php _e( 'This theme may be active on other sites in the network.' ); ?></p></div>
+					<?php
+					wp_admin_notice(
+						'<strong>' __( 'Caution:' ) . '</strong>' . __( 'This theme may be active on other sites in the network.' ),
```
Comment (joedolson) [inline src/wp-admin/network/themes.php:140]: Missing . operator fixed in commit.

File: src/wp-admin/network/themes.php [hunk 2]
```diff
@@ -135,11 +135,25 @@
...
+						)
+					);
+					?>
 					<p><?php _e( 'You are about to remove the following theme:' ); ?></p>
 				<?php else : ?>
 					<h1><?php _e( 'Delete Themes' ); ?></h1>
-					<div class="error"><p><strong><?php _e( 'Caution:' ); ?></strong> <?php _e( 'These themes may be active on other sites in the network.' ); ?></p></div>
+					<?php
+					wp_admin_notice(
+						'<strong>' __( 'Caution:' ) . '</strong>' . __( 'These themes may be active on other sites in the network.' ),
```
Comment (joedolson) [inline src/wp-admin/network/themes.php:151]: Missing . operator fixed in commit.

File: src/wp-admin/network/themes.php (hunk 1 above)
Comment (joedolson) [reply src/wp-admin/network/themes.php:140]: Also added space between </strong> and message.

File: src/wp-admin/network/themes.php (hunk 2 above)
Comment (joedolson) [reply src/wp-admin/network/themes.php:151]: Also added space between </strong> and message.

**Comment by joedolson** [comment]
In r56601

**Pull request closed without merging**